packages/harpguru-core/src/components/harp-cell/hooks/use-position-analysis/use-position-analysis.test.ts

```diff
@@ -16,20 +17,14 @@ const {
 const {
   pitchMatrix: [, , , [, , , y3x3Pitch]],
 } = activeHarpStrata
-const {
-  isActiveComplex: {
-    isActiveMatrix: [, , , [, , , y3x3IsActive]],
-  },
-} = activeHarpStrata
 
 const ourDegree = <Degree>y3x3Degree
 const ourPitch = <Pitch>y3x3Pitch
-const ourIsActive = <IsActiveIds>y3x3IsActive
 
 test('thisIsActiveId provides the isActiveId if available and undefined otherwise', () => {
   const ourCoord: YXCoord = [3, 3]
   const { thisIsActiveId } = usePositionAnalysis(ourCoord)
-  expect(thisIsActiveId).toBe(ourIsActive)
+  expect(thisIsActiveId).toBe(IsActiveIds.Inactive)
```

Comment:
`thisIsActiveId`  could probably be represented with a boolean and renamed to `thisIsActive`.

Reply:
Done